reference-stores/next/components/Search/SearchAutocomplete.js

```diff
@@ -0,0 +1,70 @@
+import { CSSTransition } from 'react-transition-group';
+import { useRouter } from 'next/router';
+import { useSearch } from 'hooks/useSearch';
+import SearchAutocompleteItem from './SearchAutocompleteItem';
+import styles from './SearchAutocomplete.module.css';
+
+const SearchAutocomplete = ({ content, show }) => {
+  const router = useRouter();
+  const { query, results } = useSearch();
+
+  const handleSearchAll = () => {
+    router.push(`/search?q=${query}`);
+  };
+
+  return (
+    content && (
+      <CSSTransition
+        in={show}
+        timeout={0}
+        classNames={{
+          enterActive: styles.enterActive,
+          exitActive: styles.exitActive,
+          enter: styles.enter,
+          exit: styles.exit,
+          enterDone: styles.enterDone
+        }}
+      >
+        <div
+          className={`
+          w-full absolute md:right-0 md:w-96 md:top-10 bg-white overflow-hidden shadow rounded-lg
```

Comment:
```suggestion
          w-full absolute lg:right-0 lg:w-96 lg:top-10 bg-white overflow-hidden shadow rounded-lg
```

Still noticed the width issue on tablet breakpoint, these changes should fix it.

Reply:
@jasonredick caught this post-merge.  Will have a new small PR to fix shortly